Add test coverage command and update README (#7647)
WordPress/wordpress-develop: README.md

npm run test:php -- --group <group name or ticket number>
```

> #### To run the coverage report

Reviewer: ```suggestion
#### Generating a code coverage report
```

> An online version can be found at https://app.codecov.io/gh/WordPress/wordpress-develop

Reviewer: ```suggestion
Code coverage reports are [generated daily](https://github.com/WordPress/wordpress-develop/actions/workflows/test-coverage.yml) and [submitted to Codecov.io](https://app.codecov.io/gh/WordPress/wordpress-develop).
```

These command run to create HTML coverage report:

```

> npm run test:test-coverage

Reviewer: ```suggestion
npm run test:coverage
```
